server/services/offender/index.js

```diff
@@ -7,7 +7,7 @@ const { Offender } = require('./responses/offender');
 const { KeyWorker } = require('./responses/keyWorker');
 const { NextVisit } = require('./responses/nextVisit');
 const { ImportantDates } = require('./responses/importantDates');
-const { TimeTable } = require('./responses/timeTable');
+const { Timetable } = require('./responses/timeTable');
```

Comment:
Can't rename one and not the filename 😛 Timetable is one word so the filename shouldn't be camelcase.

Reply:
renamed